section-09-differential-testing/model-package/requirements/diff_test_requirements_current_model.txt

```diff
@@ -0,0 +1,4 @@
+-r test_requirements.txt
+
+# Install the current prod model
+tid-regression-model==3.0.3
```

Comment:
here we are installing the version in Pypi. which I take it is the same model, but it does not bring the project together.

In section 8 we deployed to gemfury. Ideally we want to test that model. Otherwise, it becomes confusing.

Reply:
Agreed, I thought it's easier to skip the Gemfury part, but it's more consistent to keep it so shifting